EIPS/eip-3374.md

```diff
@@ -0,0 +1,57 @@
+---
+eip: 3374
+title: Predictable Proof-of-Work (POW) Sunsetting 
+author: @Query0x
+discussions-to: <tbd>
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-03-13
+---
+
+## Simple Summary
+Provides for a predictable, issuance neutral reduction of block rewards to 1 over a one year time span while reducing the uncertainties and risks associated with sudden changes to mining economics.
+
+## Abstract
+Sets the block reward to 3 ETH and then incrementally decreases it every block for 2,362,000 blocks (approximately 1 year) until it reaches 1 ETH.
+
+## Motivation
+Unnecessarily abrupt changes to the Ethereum ecosystem cause disruption and disharmony resulting in the disenfranchisement of community members while undermining stability and confidence.  While moves from Proof-of-Work to Proof-of-Stake will undoubtedly cause friction between those community members vested in either, all benefit from a measured, predictable transition.
+
+This proposal:
+
+1) Is issuance neutral.  There is no overall increase in mining rewards.
+2) Smooths the immediate impact of [EIP-1559](./eip-1559) on mining economics reducing the likeihood of reactionary responses;
```

Comment:
This change is not dependent on EIP-1559 and thus that shouldn't be mentioned.  If it is included along side EIP-1559 that is a governance decision separate from the technical specification.

Reply:
I'm not sure why you marked this as resolved.  This change is not dependent on 1559 (on a technical level) and thus shouldn't be referencing it.  If you want to propose this be included along side 1559 that is totally fine, and this line may be an argument in favor of that, but it isn't appropriate for the technical specification.

Keep in mind that this EIP could be applicable without EIP-1559 such as in the face of a change in block times, or a sharp drop in price, etc.